packages/utils/src/order/status.ts

```diff
@@ -29,3 +29,31 @@ export enum OrderStatus {
    */
   REQUIRES_ACTION = "requires_action",
 }
+
+/**
+ * @enum
+ *
+ * The order change's status.
+ */
+export enum OrderChangeStatus {
```

Comment:
I am just wondering, is utils the best place for things like enums? Wouldn't this be better placed as part of a module (and exported from it in some way)? Alternatively, it can be part of the API definition, but like this It feels like we are leaking domain models to the public

Reply:
These are public exposed properties of the module, so I do not consider this a "leak".
We don't have a clean solution to place enums, that behave like a type, but can't be added to the types package.
One shouldn't need the module to have access to their public types.

Reply:
I think discussing this here will require several paragraphs so we can talk about the different approaches in defining module definitions some other time. But either way, putting enums in utils is a bit dirty in my opinion. The types package could be a better place where we expose enums, WDYT?

Reply:
`@medusajs/types` should be strictly a dev dependency, as enums will generate an object, they can't be referenced at the code. They won't exist in production.
That's the main reason it is not placed there.

Reply:
Got it, thanks @carlos-r-l-rodrigues . I guess we can look for a better place later on. My previous experience was that the sdk would carry types, enums, and wrappers around the backend API, but I think here we want them to be more independent.